lib/Account.php

```diff
@@ -150,8 +150,8 @@ private function serializeAdditionalOwners($legalEntity, $additionalOwners)
     }
 
     /**
-     * @param null|array|string $id The ID of the account to retrieve, or an
-     *     options array containing an `id` key.
+     * @param null|array|string $id the ID of the account to retrieve, or an
```

Comment:
Surprised they don't like the leading upper case letter here, especially as the rule is about not ending with a dot
Nitpicky but not sure it looks nicer in IDEs to be honest